### PR TITLE
feat: Add navigation menu

### PR DESCRIPTION
**Pull request by Alystrasz**
For next year, @sbliudze (general chair) wants a navigation menu [just like SEFM 2024 does](https://sefm-conference.github.io/2024/callforpapers/); this thus imports navigation component and related style from [minima theme](https://github.com/sefm-conference/2024/tree/main/_sass).

* For the `2025` subsite, I removed the banner image from each page file (no more duplication), and put it in a `header.html` template file;
* This template file is only included for `/2025` pages (= other websites should not be impacted by this PR);
* To be included in the navigation bar, a page must include the following header:
```text
---
title: Accepted papers
menu_show: true
order: 1
---
```

I also slightly modified the menu to always appear on screen, no matter the scroll level.

##### Before

image

##### After

image

### CODE REVIEW

**Comment by basvdheuvel** [comment]
Looks and works great! It may be nice to implement submenu's in the future, but that can be a different PR.
I just changed the variables used in SCSS to use the defaults from our own theme (`jekyll-theme-primer`).
Please check and confirm, then I'll merge.

File: assets/css/style.scss [hunk 1]
```diff
@@ -185,8 +187,8 @@ $spacing-unit:     30px !default;
   position: absolute;
   bottom: 10px;
   left: 14px;
-  color: white;
-  background: #2b2727b0;
+  color: $text-white;
+  background: $bg-gray-dark;
```
Comment (Alystrasz) [inline assets/css/style.scss:191]: I added both `banner` and `banner-credit` classes to style (they do not belong to original template style), so they don't necessarily need to inherit template style; particularly regarding the `background` change, I wanted to find a level of opacity that somehow matches the title one, while still keeping the photo credits readable.

##### Without your proposed change

image

##### With your proposed change

image

**Review by Alystrasz — COMMENTED** [review]
Thanks for inserting template values!
I have a few remarks regarding elements that should stay the way I proposed, then we're good.

File: assets/css/style.scss [hunk 2]
```diff
@@ -44,10 +44,10 @@ $spacing-unit:     30px !default;
   letter-spacing: 0px;
   margin-bottom: 0;
   float: left;
-  padding-right:32px;
+  padding-right:$spacer-5;
 
-  font-family: "Roboto", sans-serif;
-  font-weight: 300;
+  font-family: $body-font;
```
[the remark this answers is not in the document]
Comment (basvdheuvel) [reply assets/css/style.scss:49]: That's fair, but I do think it's a bit too thin in your version. But go ahead and fix that.

File: assets/css/style.scss (hunk 1 above)
Comment (basvdheuvel) [reply assets/css/style.scss:191]: Right, I didn't spot the opacity, go ahead and revert that.

Comment (Alystrasz) [reply assets/css/style.scss]: Reverted in 4552c5b4388bec73d3bcec1dfdc8db56e823ec77.

File: assets/css/style.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (Alystrasz) [reply assets/css/style.scss:49]: To still differentiate title from menu items, we could go the other way and make the title bolder, e.g. `font-weight: 500;`.
WDYT?

[the remark this answers is not in the document]
Comment (basvdheuvel) [reply assets/css/style.scss:49]: Good idea. We have then three options:

`$font-weight-light`:
image

`$font-weight-semibold`:
image

`$font-weight-bold` (my preference):
image

[the remark this answers is not in the document]
Comment (Alystrasz) [reply assets/css/style.scss:49]: I also think `$font-weight-bold` is the most suitable option!

**Comment by basvdheuvel** [comment]
Anything else before I merge?

**Comment by Alystrasz** [comment]
> Anything else before I merge?

I just ran the changes locally, seems good to me.

**Pull request merged**